ex1/exercise_1_x_mix_drix.py

```diff
@@ -95,15 +99,17 @@ def play_game(self, n_moves: int):
         iterator = iter(self._user_input_generator)
         for i in range(n_moves):
             # Told for the user who is turn, collect a row and column from the user and try to put a play token there.
-            print(f"Now it's the turn of the player is play with {self._current_player.value}")
+            print(f"Your turn")
             row, column = next(iterator)
             self._board.play_move(row, column, self._current_player)
             print(self._board)
 
-            # Check if the current player in won.
-            if self._board.is_winning(self._current_player) or " " not in str(self._board):
-                break
-            self.change_player()
+            # The computer random choice
+            print("The computer turn...")
+            row, column = (random.randint(0, 2), random.randint(0, 2))
+            self._board.play_move(row, column, self._computer_player)
```

Comment:
Very good starting point, implement the suggestions you sent me over email.
I will give you feedback only once the code is pushed.